app/controllers/two_factor_authentication/otp_verification_controller.rb

```diff
@@ -1,7 +1,9 @@
 module TwoFactorAuthentication
-  class OtpVerificationController < DeviseController
+  class OtpVerificationController < ApplicationController
     include TwoFactorAuthenticatable
 
+    skip_before_action :handle_two_factor_authentication
```

Comment:
I think it's clearer to explicitly skip it here? Rather than try to complicate the "skip if" logic inside the helper?

We could add a `return if otp_verificiation_controller?` method or something, but then we'd have to add that method to all the other controllers, when all we want is to skip the method here, so I think adding the skip here makes sense?